src/components/item/ItemMain.tsx

```diff
@@ -62,7 +61,7 @@ const ItemMain = ({ id, children, item }: Props): JSX.Element => {
             direction="rtl"
           >
             <Typography variant="h6">
-              {translateBuilder(BUILDER.ITEM_CHATBOX_TITLE)}
+              {translateBuilder(`"${item.name}"`)} chat
```

Comment:
Is it correct that if my item is named `lesson 1` you are translating `lesson 1` and then concatenating it with ` chat` ?
I think the name of the item does not need to be translated. But the `chat` part does. For this you should use [i18next Interpolation feature](https://www.i18next.com/translation-function/interpolation). 
Some examples of this would be:
```ts
{translateBuilder(BUILDER.ITEM_CHATBOX_TITLE, { name: item.name })}
```
We will need to update `ITEM_CHATBOX_TITLE` string to be `"{{name}}" chat`.
It will then be translated in different languages: 
french: `Discussion pour "{{name}}"` for example.